src/main/update/launcher-update.ts

```diff
@@ -69,6 +69,10 @@ export interface IUpdateOptions {
 }
 
 export async function update(update: Update, listeners: IUpdateOptions) {
+  if (!getConfig("UseUpdate", true)) {
```

Comment:
```suggestion
  if (!getConfig("UseUpdate", process.env.NODE_ENV === "production")) {
```

Just a touch to disable updates during development.

Reply:
Gooooood